api/v1/clustercatalog_types.go

```diff
@@ -34,6 +34,14 @@ const (
 
 	AvailabilityModeAvailable   AvailabilityMode = "Available"
 	AvailabilityModeUnavailable AvailabilityMode = "Unavailable"
+
+	// Serving condition types
```

Comment:
nit: ~Serving~ Condition types?

Reply:
Done 👍🏽 
I pushed a change to address Joe's feedback too can I get a re-lgtm please 🙏🏽